config/config.go

```diff
@@ -39,6 +39,9 @@ func Default() *Config {
 	conf.Locust.Host = "127.0.0.1"
 	conf.Locust.Port = 5557
 
+	conf.Ably.ConnectionTimeout = 10 * time.Second
+	conf.Ably.RequestTimeout = 30 * time.Second
```

Comment:
I think these should default to 4s and 10s respectively to match the default Ably SDK values